app/utils/NextPage.scala

```diff
@@ -40,6 +41,17 @@ trait NextPage[A, B] {
 
 object NextPage {
 
+  implicit val haveYouRegisteredTrust: NextPage[HaveYouRegisteredTrustId.type,
+    models.sa.trust.HaveYouRegisteredTrust] = {
```

Comment:
Consider not fully qualifying `HaveYouRegisteredTrust`. It is this way due to template constraints but can be changed